src/lib/actions.tsx

```diff
@@ -87,26 +92,25 @@ const submitUserMessage = async (userInput: string, focus: Focus) => {
     const initialMessages: Message[] = [
       systemMessage, // system message with full document info
       {
-        id: userMsgId,
+        id: nanoid(), // new id for the user message
```

Comment:
i tried to do this but i could not get a type for passing `aiState` and i didn't want to do `any` nor did i want to open two `getMutableAIState`

Reply:
😒 yes i just said i tried to do that and the typing was not available because `ai/rsc` does not export `MutableAIState<T>` which is the type of `aiState`